code/game/objects/effects/decals/cleanable/misc.dm

```diff
@@ -10,6 +10,7 @@
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "ash"
 	mergeable_decal = FALSE
+	turf_loc_check=FALSE

```

Comment:
Remove this from the type definition, we still want to prevent ash decals spawning whereever they want in general. Instead, make the signature of `/obj/effect/decal/cleanable/ash/Initialize()` into something like `/obj/effect/decal/cleanable/ash/Initialize(/obj/effect/decal/cleanable/Initialize(mapload, list/datum/disease/diseases, force = FALSE)` and in Initialize() do something like `if (force) src.turf_loc_check = force` before `. = ..()`. Feel free to use whichever var names you feel like though. 

Then in morgue.dm replace `new/obj/effect/decal/cleanable/ash(src)` with `new/obj/effect/decal/cleanable/ash(src,force=TRUE)`